src/librustc/metadata/creader.rs

```diff
@@ -49,15 +49,16 @@ impl<'a, 'v> visit::Visitor<'v> for CrateReader<'a> {
 }
 
 fn dump_crates(cstore: &CStore) {
-    debug!("resolved crates:");
```

Comment:
Yeah this was currently intentional but I wouldn't mind changing it back. Whenever someone's having weird behavior with loading crates I typically ask them to set `RUST_LOG=rustc::metadata::creader,rustc::metadata::loader` to get info like this, but `debug!` means it's all compiled out by default.

Along those lines I'd somewhat prefer to have it be `info!` but I don't have much of a preference either way